rofl-containers/src/containers.rs

```diff
@@ -60,5 +82,39 @@ pub async fn start() -> Result<()> {
         .current_dir("/etc/oasis/containers")
         .spawn()?;
 
+    // Purge environment file as it is not needed anymore.
+    fs::remove_file("/run/podman/env")?;
+
     Ok(())
 }
+
+static GLOBAL_ENVIRONMENT: LazyLock<Environment> = LazyLock::new(Environment::new);
+
+/// Management of environment variables to expose to the compose file.
+pub fn env() -> &'static Environment {
+    &GLOBAL_ENVIRONMENT
+}
+
+/// Management of environment variables to expose to the compose file.
```

Comment:
It would also be useful in some cases to expose the configured external IP of the machine on which the rofl app is running as an environment variable.

Reply:
I agree. It would need to be explicitly configured. We can also add/implement this at a later time, but I think it might be useful in some cases.

Reply:
Added. The scheduler reads it from its config and then forwards it to apps via the proxy configuration label. Then we expose it to containers here. If it is not configured in the scheduler config, the value is not exposed.